INTRO.md

```diff
@@ -4,15 +4,6 @@ This is the official Filecoin protocol specification. It is a work in progress.
 
 This collection of pages specify the protocols comprising the Filecoin network. The goal of these specs is to provide sufficient detail that another implementation, written using only this document as reference, can be fully compatible with `go-filecoin` peers, but this is still a work in progress.
 
-
-### Style
```

Comment:
I don't think we should remove this. We should adhere to this strctly, and address the (many) places that we don't use this correctly.

Reply:
sorry, wasn't meant to be dropped, but moved. then forgot to add the file. will add it now.

Reply:
Added. The reasoning for splitting that out were the following two:

1. I want to add more (like the just added hints in #259 ), making this a lot more verbose and though
2. it is often said in the beginning of IETF RFCs, I think it hinders the reading flow here.

I would see the argument about having it as a sup-entry under "Overview" on the left (rather than at "SPEC" at the end) but I'd argue, it is less important for usual reader.